tests/features/joinup_group/short_id.feature

```diff
@@ -0,0 +1,130 @@
+@api @terms
+Feature:
+  As an owner of a group
+  In order to make the URL of my group easier to read
+  I need to be able to provide a short ID to my group.
```

Comment:
This describes that the short ID is intended to be used in the URL but the test doesn't look at the URL at any point. Also in the PR I cannot find the code that would actually affect the URL aliases?

Is the changing of the URL intended for a followup ticket or is this handled here? I cannot tell without having access to Jira. It would be strange to deploy this on production if the URLs are not actually using this short ID.

Reply:
The URL setup is not fixed. There is a follow up for this.

Reply:
Sorry I also cannot help you right now without Jira but I know there is a follow up.

Reply:
:+1: